packages/reports/src/Stores/Modules/Profit/Helpers/__tests__/format-response.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { formatProfitTableTransactions, TTransaction } from '../format-response';
+import { ActiveSymbols } from '@deriv/api-types';
+
+let mockProfitTableTransactionData: TTransaction;
+describe('formatProfitTableTransactions', () => {
+    beforeEach(() => {
+        mockProfitTableTransactionData = {
+            app_id: 11780,
+            buy_price: 1,
+            contract_id: 55113405821,
+            contract_type: 'CALL',
+            duration_type: 'minutes',
+            longcode:
+                'Win payout if Volatility 100 (1s) Index is strictly higher than entry spot at 15 minutes after contract start time.',
+            payout: 1.96,
+            purchase_time: 1700539661,
+            sell_price: 0.61,
+            sell_time: 1700539816,
+            shortcode: 'CALL_1HZ100V_1.96_1700539661_1700540561_S0P_0',
+            transaction_id: 111189668101,
+            underlying_symbol: '1HZ100V',
+        };
+    });
+
+    it('should return correct formatted object', () => {
+        const currency = 'USD';
+        const active_symbols: ActiveSymbols = [];
+        const returnValue = formatProfitTableTransactions(mockProfitTableTransactionData, currency, active_symbols);
+        expect(returnValue).toEqual({
+            app_id: 11780,
```

Comment:
Many pairs in this object are the same as in `mockProfitTableTransactionData`. If you want to avoid duplications, you can use spread operator and rewrite or add some new fields. Smth like:
`.toEqual({...mockProfitTableTransactionData, display_name:'',})`

Reply:
Ignore if irrelevant